src/rct2.c

```diff
@@ -301,21 +301,22 @@ void rct2_draw(rct_drawpixelinfo *dpi)
 	gCurrentDrawCount++;
 }
 
-static uint32 _lastFPSUpdateTicks;
-static uint32 _lastFPSTicks;
-static float _currentFPS;
+static time_t _lastSecond;
+static int _currentFPS;
+static int _frames;
 
-static float rct2_measure_fps()
+static void rct2_measure_fps()
 {
-	uint32 currentTicks = SDL_GetTicks();
-	if (currentTicks - _lastFPSUpdateTicks > 500) {
-		_lastFPSUpdateTicks = currentTicks;
+    _frames++;
```

Comment:
Fixed that. Is it not possible to just plan some point in time where the majority of contributors agree to convert all files at once?